lib/BrowserUtils.js

```diff
@@ -3,10 +3,15 @@ const fs = require('fs');
 const logger = require('./Logger');
 const path = require('path');
 const Utils = require('./Utils');
+const wdioUi5 = require('./wdioUi5-index')
 
 // Singleton
 module.exports = class BrowserUtil extends Utils {
     static _instance;
+    path = {
```

Comment:
would also be helpful to have a comment here why `electron` is part of BrowserUtils.js (and not NativeUtils)

Reply:
I am going to write it in Readme.